### ISSUE TITLE
Building of fwupd  v 1.0.8 fails with error: Invalid version of dependency, need 'fwup' ['>= 10'] found '9'

### ISSUE THREAD

**Issue by maxim27** [hughsie/fwupd#566]
I am trying to build version 1.0.8 of fwupd (on opensuse tumbleweed) but the build fails with the following error:
`Native dependency fwup found: NO found '9' but need: '>= 10'
 meson.build:198:2: ERROR:  Invalid version of dependency, need 'fwup' ['>= 10'] found '9'.`

Should not fwup be part of fwupd?

**Comment by hughsie** [comment]
No, fwupdate is a different project to fwupd. We're actually talking about merging them in the future as I agree it's confusing: https://github.com/rhboot/fwupdate/issues/117
